NEWS

```diff
@@ -29,6 +29,14 @@ Changes
 
 * Add a new test dependency of testscenarios. (Robert Collins)
 
+* ``addCleanup`` can now only be called during a test run.
```

Comment:
nit - perhaps 
'called within test.run()' I realise thats technically ambiguous for twisted with its non-generator async idiom, but I think its still clearer?